spec/features/admin_claim_check_spec.rb

```diff
@@ -93,6 +93,20 @@
       expect(page).to have_content("Claim decision")
     end
 
+    scenario "User can see completed checks" do
+      qualification_check = build(:check, name: "qualifications")
+      claim_with_checks = create(:claim, :submitted, checks: [qualification_check, build(:check, name: "employment")])
+      visit admin_claim_checks_path(claim_with_checks)
+
+      expect(page).to have_content("Check qualification information Completed")
+      expect(page).to have_content("Check employment information Completed")
+
+      click_on "Check qualification information"
+      expect(page).to have_content("Checked by #{user.full_name}")
+      expect(page).to have_content(I18n.l(qualification_check.created_at))
```

Comment:
Can we set an explicit check date and verify that on the page? We already show the submitted at date on the page so we could end up with a false positive.